packages/sveltekit/src/vite/sourceMaps.ts

```diff
@@ -27,6 +28,18 @@ type Sorcery = {
 // and we only want to generate a uuid once in case we have to fall back to it.
 const releaseName = detectSentryRelease();
 
+let sourceMapSetting: {
+  updatedSourceMapSetting?: boolean | 'inline' | 'hidden';
+  previousSourceMapSetting?: UserSourceMapSetting;
+} = { previousSourceMapSetting: undefined, updatedSourceMapSetting: undefined };
+
+/**
+ * For mocking the value of `sourceMapSetting` in tests
+ */
+export const __setSourceMapSettingForTest = (value: typeof sourceMapSetting): void => {
+  sourceMapSetting = value;
+};
+
```

Comment:
m: Can we use ? `vi.mock('vite')` or something along these lines to avoid having to expose this? My gut feeling tells me there must be a cleaner way to solve this but I might be missing something 😅

Reply:
I tried it with a mock but I was unsuccessful :/
Looked like this:
```js
      vi.mock('../../src/vite/sourceMaps', async importOriginal => ({
        ...((await importOriginal()) as { makeCustomSentryVitePlugins: () => void }),
        sourceMapSetting: {
          previousSourceMapSetting: 'enabled', // <--- this is the variable we want to change in the tests
          updatedSourceMapSetting: undefined,
        },
      }));
```

But I guess because it's a variable defined at the root of the file, set and read in different places, the mock does not provide the correct values :/